### ISSUE TITLE
a11y audit

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wp20.wordpress.net#11]
Issues to be fixed listed in comments below.

**Comment by iandunn** [comment]
Related #16

**Comment by iandunn** [comment]
It sounds like [the non-breaking space workaround](https://github.com/WordPress/wp20.wordpress.net/pull/44) might not be [read smoothly by Voiceover](https://github.com/WordPress/wporg-mu-plugins/pull/386#issuecomment-1501454080)? It was merged in that PR, though, so may be good enough for this site too.

**Comment by adamwoodnz** [comment]
> It sounds like [the non-breaking space workaround](https://github.com/WordPress/wp20.wordpress.net/pull/44) might not be [read smoothly by Voiceover](https://github.com/WordPress/wporg-mu-plugins/pull/386#issuecomment-1501454080)? It was merged in that PR, though, so may be good enough for this site too.

Just tested and found [the same thing as Kelly](https://github.com/WordPress/wporg-mu-plugins/pull/386#issuecomment-1502133394), the `nbsp;` isn't read by Voiceover for me.

**Comment by adamwoodnz** [comment]
![Screen Shot 2023-04-11 at 5 24 45 PM](https://user-images.githubusercontent.com/1017872/231064010-c87a2b5d-b326-4947-8981-360fd7aa00b6.jpg)

@marko-srb what d you think about adding underlines to these links? Alternatively we could use a darker shade of blueberry, just for these.

**Comment by adamwoodnz** [comment]
The map markers contain buttons which fail

![Screen Shot 2023-04-11 at 5 28 37 PM](https://user-images.githubusercontent.com/1017872/231064767-9fde7adc-ef67-4745-8ce0-3d16e779afd3.jpg)

![Screen Shot 2023-04-11 at 5 30 39 PM](https://user-images.githubusercontent.com/1017872/231065041-5060673e-0e26-4310-a5bb-4b5c054c35ce.jpg)

The screenshot above shows the elements do have an `aria-describedby` attribute, but that isn't useful, and we need to implement one of the fixes listed. @iandunn maybe we can change the `aria-label` or `aria-labelledby` properties via the API?

**Comment by adamwoodnz** [comment]
Meetup Group titles should be H2 

![Screen Shot 2023-04-11 at 5 43 38 PM](https://user-images.githubusercontent.com/1017872/231067097-0957fe98-3be2-4086-be2f-b8d505640212.jpg)

**Comment by adamwoodnz** [comment]
Swag item titles should be H2

![Screen Shot 2023-04-11 at 5 51 07 PM](https://user-images.githubusercontent.com/1017872/231068290-3b28324d-aba2-4d8a-938f-3b2a8e5c36fa.jpg)
